zilliqa/src/message.rs

```diff
@@ -245,6 +245,15 @@ impl QuorumCertificate {
     }
 }
 
+impl Display for QuorumCertificate {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        write!(f, "QC hash: {}, ", self.compute_hash())?;
+        write!(f, "QC signature: [..], ")?;
```

Comment:
The signature is a fixed length - Is there a reason we don't display it?

Reply:
It doesn't really offer useful information IMO. And if you do display it, it fills your terminal up with stuff you likely don't care about. Debug will display it though ofc